lib/grammars.coffee

```diff
@@ -253,6 +253,11 @@ module.exports =
       command: "rails"
       args: (context) -> ['runner', context.filepath]
 
+  Rust:
+    "File Based":
+      command: "bash"
+      args: (context) -> ['-c', "rustc " + context.filepath + " -o /tmp/rs.out && /tmp/rs.out"]
```

Comment:
Pardon my abrasiveness, but does this actually run? The entire " -o /tmp/rs.out && /tmp/rs.out" will get passed as an argument to rustc instead of being separate pieces.

Also, this only works for *nix OSes. /cc @erran.